doc/interfaces/kvaser.rst

```diff
@@ -39,6 +39,11 @@ in the ``recv`` method. If a message does not match any of the filters,
 ``recv()`` will return None.
 
 
+ISO/Non-ISO CAN FD Mode
+-----------------------
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~
```

This heading is a bit too large. 
![image](https://github.com/hardbyte/python-can/assets/52598363/6eb87556-8c69-41a2-8351-b3edba0773d1)
Also, i think the docstring for the new parameter is sufficient,  this paragraph does not add any additional information.